docs/root/intro/deployment_types/service_to_service.rst

```diff
@@ -14,7 +14,7 @@ Service to service egress listener
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This is the port used by applications to talk to other services in the infrastructure. For example,
-http://localhost:9001. HTTP and gRPC requests use the HTTP/1.1 *host* header or the HTTP/2 or HTTP/3
+*http://localhost:9001*. HTTP and gRPC requests use the HTTP/1.1 *host* header or the HTTP/2 or HTTP/3
```

Comment:
im not sure what the problem with the link is, but i dont think italicizing it helps any

Reply:
@phlax Hi, Thanks for review. My Point is, placing URLs here will create confusion as it is not configuration sections where user can use it while configuring it. Moreover also it also helps to view in consistence similar to below section as https://www.envoyproxy.io/docs/envoy/latest/intro/deployment_types/service_to_service#service-to-service-ingress-listener

Reply:
for  italicizing, i just checked section https://www.envoyproxy.io/docs/envoy/latest/intro/deployment_types/service_to_service#service-to-service-ingress-listener and its source code for removing urls. We can just remove urls without  italicizing etc.

Reply:
im trying to get rid of the use of italics, its one of the things that has made the docs really hard to read

not sure about removal, but if you want to update the PR to remove, im happy to read the docs through properly

Reply:
Ok.

Reply:
@phlax just on the closure side of this PR. Would you please help me understand what is the meaning of read the docs here ?

Reply:
is that https://readthedocs.org/  ? Do we have readthe docs for envoyproxy as well and if yes, how do we do code changes here ?

Reply:
we use the RTD (readthedocs) theme, we dont host with them tho

Reply:
Ok. Got it. Thanks for quick response

Reply:
Before reaching to conclusion codewise, i would like to highlight below URLs of ONAP community, They follow RTD. please suggest if it is ok to take as example here as well.
https://docs.onap.org/en/latest/guides/onap-user/onap-portal-user/onap-portal-user.html#access

Reply:
as i said, there is no benefit to using italics, it is semantically wrong and confusing

italics are for emphasis, using it elsewhere renders that meaningless and does nothing for code or urls

i think the best thing is to leave this as it is

Reply:
Okay.